packages/cli/src/common/filenames.ts

```diff
@@ -3,3 +3,7 @@ import * as inflected from 'inflected'
 export function classNameToFileName(name: string): string {
   return inflected.dasherize(inflected.underscore(name))
 }
+
+export function resourceNameToClassName(name: string): string {
+  return inflected.classify(inflected.underscore(inflected.parameterize(inflected.tableize(name))))
```

Comment:
Actual question, why not `classify` directly?

Reply:
`classify` itself will transform name like `test-command` to  `Test-Command`, 
`parameterize` is used to make sure that spaces are changed to `-`
And since classify is not removing `-`,  `underscore` is used to replace `-` to `_`

I think `tableize` can be removed since it looks like it is not needed

Reply:
@NickSeagull Just checked once more, `tableize` is needed too. It is adding underscore between words `Inflector.tableize('RawScaledScorer')  // => 'raw_scaled_scorers'`
Without it, `testCommand` will transform to `testcommand` by `parameterize` method

Reply:
I see, thanks!